src/sink/map_err.rs

```diff
@@ -0,0 +1,30 @@
+use sink::Sink;
+
+use {Poll, StartSend};
+
+/// Sink for the `Sink::map_err` combinator.
+#[must_use = "sinks do nothing unless polled"]
+pub struct MapErr<S, F> {
```

Comment:
Could this be called `SinkMapErr` for consistency? (also the comment above this may need updating)